Stripe/STPPaymentContextAmountModel.m

```diff
@@ -35,28 +35,49 @@ - (instancetype)initWithPaymentSummaryItems:(NSArray<PKPaymentSummaryItem *> *)p
     return self;
 }
 
-- (NSInteger)paymentAmountWithCurrency:(NSString *)paymentCurrency {
+- (NSInteger)paymentAmountWithCurrency:(NSString *)currency shippingMethod:(PKShippingMethod *)shippingMethod {
```

Comment:
I initially added a new set of tests for `STPPaymentContextAmountModel`, but @bdorfman noted that there was some overlap in [STPPaymentContextTest](https://github.com/stripe/stripe-ios/blob/bg-shipping-pc/Tests/Tests/STPPaymentContextTest.m) [0], so I've instead just added tests to `STPPaymentContextTest`.

[0] https://github.com/stripe/stripe-ios/pull/451#discussion_r79211644